serial_gui.py

```diff
@@ -1,10 +1,26 @@
 import tkinter
+from tkinter.constants import END
+from typing import Text
 
 #from serial_connection import write_to_serial
 from mock_arduino import mock_arduino_response
 
 main_window = tkinter.Tk()
 
-serial_connection_button = tkinter.Button(main_window, text = "Hello", command = mock_arduino_response)
-serial_connection_button.pack()
-main_window.mainloop()
+def send_button_click():
+    output_box.configure(text=mock_arduino_response(input_box.get("1.0", 'end-1c')))
+    print(input_box.get("1.0", 'end-1c'))
```

Comment:
this print line seems to act as some debug, so can be removed

Reply:
I tried removing this and it broke, so it's staying in :D